add docs regarding pods launchpad (#453)
localstack/docs: content/en/user-guide/tools/cloud-pods/launchpad/index.md

---
title: "Cloud Pods Launchpad"
weight: 10
categories: ["LocalStack Pro", "Tools"]
description: >
  Our Cloud Pods Launchpad offers the easiest way of sharing and injecting cloud pods directly from GitHub.
---

## Adding a badge to your repository

If you want to add a badge to your README that links to the Cloud Pod Launchpad, you can use the following markdown snippet:

```markdown
[![LocalStack Pods Launchpad](https://localstack.cloud/gh/pod_badge.svg)](https://app.localstack.cloud/launchpad?url=url_of_your_pod)
```

Please note that you have to add your `url_of_your_pod` as the `url` query parameter to the URL.
For example if you are hosting the pod in the same repository, just use the URL pointing to the raw pod zip.

Check out a proper example <a href="https://github.com/localstack/cloud-pod-badge" target="_blank">on this GitHub repository</a>.

> ## Security concerns regarding the launchpad
> 
> Please consider that the launchpad takes an arbitrary URL as input and passes it onto your LocalStack instance to inject the pod. 
> This may lead to unintended side effects, as the URL that is passed to the launchpad is not validated in any way. 
> 
> We also display the URL on the UI, so we recommend to only use the launchpad with trusted URLs.

Reviewer: Maybe use an alert box since this is a security note?

```suggestion
{{< alert title="Warning" color="warning" >}}
Please consider that the launchpad takes an arbitrary URL as input and passes it onto your LocalStack instance to inject the pod. 
This may lead to unintended side effects, as the URL that is passed to the launchpad is not validated in any way. 

We also display the URL on the UI, so we recommend to only use the launchpad with trusted URLs.
{{< /alert >}}
```

Author: thanks a lot @viren-nadkarni !
awesome proposal :)

---
title: "Cloud Pods Launchpad"
weight: 10
categories: ["LocalStack Pro", "Tools"]
description: >
  Our Cloud Pods Launchpad offers the easiest way of sharing and injecting cloud pods directly from GitHub.
---

## Adding a badge to your repository

If you want to add a badge to your README that links to the Cloud Pod Launchpad, you can use the following markdown snippet:

```markdown

> [![LocalStack Pods Launchpad](https://localstack.cloud/gh/pod_badge.svg)](https://app.localstack.cloud/launchpad?url=url_of_your_pod)

Reviewer 2: ```suggestion
[![Launch LocalStack Cloudpod](https://localstack.cloud/gh/launch-pod-badge.svg)](https://app.localstack.cloud/launchpad?url=url_of_your_pod)
```

Author: adjusted the other repos accordingly (had to change the name of the badge file) + updated with a screenshot of a repo + badge

---
title: "Cloud Pods Launchpad"
weight: 10
categories: ["LocalStack Pro", "Tools"]
description: >
  The LocalStack Cloud Pods Launchpad offers the easiest way of sharing and injecting cloud pods directly from your repositories or via a simple URL.
---

## Overview

> For more general information about cloud pods, please visit the [Cloud Pods Overview]({{< ref "cloud-pods" >}}). <br>

Reviewer 3: ```suggestion
For more general information about Cloud Pods, please visit the [Cloud Pods Overview]({{< ref "cloud-pods" >}}). <br>
```

The LocalStack Cloud Pods Launchpad allows you to quickly share and inject cloud pods into a LocalStack instance.

## Creating your sharable link

> Navigate to [app.localstack.cloud/launchpad](https://app.localstack.cloud/launchpad) where you can create a sharable link for your pods.

Reviewer 3: Clarification question: is the badge hard-coded, right? What if we want to have multiple badged in a repo? How can we distinguish them?

Author: As of right now the badges would be identical - there's no good way of customizing these without relying on shield.io's creation.

Author: How would you like to customize/distinguish between the badges? @giograno

Author: In case we want to add dynamically labeled badges, I think we should try to get our logo into [simple icons](https://github.com/simple-icons/simple-icons), to be used as outlined in shields.io docs here: https://github.com/badges/shields/blob/master/doc/logos.md#simpleicons

also @thrau
